src/libs/SaveJSONFile.ts

```diff
@@ -0,0 +1,6 @@
+import fs from 'fs';
+
+export default function saveJsonFile(fileName: string, records: object[]) {
+  const jsonString = JSON.stringify(records);
+  fs.writeFileSync(fileName, jsonString);
+}
```

Comment:
Maybe instead of specific function to save JSON file, we can create one generic function for writing string to file. It might be useful in the future when we want to implement import SQL, XML, etc...

For JSON stringify, you can use `JSON.stringify(records, undefined, 2);`, this will provide properly indentation.

Reply:
Let me refactor to be a generic function instead and format the JSON.